src/lavfmuxer.cpp

```diff
@@ -101,6 +110,14 @@ lavfmuxer::lavfmuxer(const char *format, uint32_t audiostreammask, mpgfile &mpg,
 	}
       }
       s->time_base = codec->time_base;
+#if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(57, 14, 0)
+      // Use Parameters instead of Codec, necessary to get correct results, prevent crash and correct display of the output of av_dump
+      ret = avcodec_parameters_from_context(s->codecpar, codec);
+      if (ret < 0) {
+        av_log(NULL, AV_LOG_ERROR, "Failed to copy encoder parameters to output stream #%u\n", s->id);
+        return;
+      }
+#endif      
```

Comment:
And this even more cosmetic - could you remove the trailing whitespaces?

Reply:
I confirm with you to have a common spacing but I seem to be just blind to see, the correct one to be use.

Reply:
Line 120 contains `#endif      `, it just did light up in `git gui` on my side.

Reply:
Thanks for the hint.